worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/FileWriter.java

```diff
@@ -440,4 +446,12 @@ public void notifyNonCriticalError(String mountPoint, DiskStatus diskStatus) {}
   public PartitionType getPartitionType() {
     return partitionType;
   }
+
+  public void setShuffleKey(String shuffleKey) {
+    this.shuffleKey = shuffleKey;
+  }
+
+  public void setStorageManager(StorageManager storageManager) {
```

Comment:
Better to pass `shuffleKey` and `storageManager` as constructor parameters instead of adding this set methods.